vendor/printenvz/binding.gyp

```diff
@@ -0,0 +1,23 @@
+{
+  "targets": [
+    {
+      "target_name": "printenvz",
+      "type": "executable",
+      "sources": [
+        "src/printenvz.cc"
+      ],
+      "include_dirs": [],
+      "cflags": ["-std=c++11"],
+      "cflags_cc": ["-std=c++11"],
+      "conditions": [
+        ["OS=='mac'", {
+          "xcode_settings": {
+            "OTHER_CPLUSPLUSFLAGS": ["-std=c++11", "-stdlib=libc++"],
+            "OTHER_LDFLAGS": ["-stdlib=libc++"],
+            "MACOSX_DEPLOYMENT_TARGET": "10.7"
+          }
+        }]
+      ]
+    }
+  ]
+}
```

Comment:
Per https://github.com/github/security-reviews/issues/299

You might want to consider adding certain flags like the ones from https://github.com/desktop/desktop-trampoline/pull/1

Reply:
Done!